lib/WebService/MinFraud/Record/CreditCard.pm

```diff
@@ -35,6 +42,12 @@ has is_prepaid => (
     coerce  => BoolCoercion,
 );
 
+has type => (
+    is        => 'ro',
+    isa       => Maybe [ Enum [qw( charge credit debit )] ],
```

Comment:
That's a good point, I probably should relax it to a string.

2016-01-19 15:45 GMT-07:00 Gregory Oschwald <notifications@github.com>:

> In lib/WebService/MinFraud/Record/CreditCard.pm
> <https://github.com/maxmind/minfraud-api-perl/pull/10#discussion_r50188377>
> :
>
> > @@ -35,6 +42,12 @@ has is_prepaid => (
> >      coerce  => BoolCoercion,
> >  );
> >
> > +has type => (
> > +    is        => 'ro',
> > +    isa       => Maybe [ Enum [qw( charge credit debit )] ],
>
> Not sure about this validation. Do we want this to fail if we decide to
> add a new type?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maxmind/minfraud-api-perl/pull/10/files#r50188377>.
>